datafusion/common/src/column.rs

```diff
@@ -71,7 +71,11 @@ impl Column {
         }
     }
 
-    fn from_idents(idents: &mut Vec<String>) -> Option<Self> {
+    /// Create a Column from multiple normalized identifiers
+    ///
+    /// For example, `foo.bar` would be represented as a two element vector
+    /// `["foo", "bar"]`
+    pub fn from_idents(mut idents: Vec<String>) -> Option<Self> {
```

Comment:
i made this public and cleaned up the signature so I could reuse it when dealing with the changes to `USING` planning